### PR TITLE
fixes issue #973

### PR DESCRIPTION
**Pull request by gracekarina**
Set correct integer value #973

### CODE REVIEW

File: modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java [hunk 1]
```diff
@@ -57,6 +57,20 @@
...
+        options.setResolve(true);
+        final OpenAPI openAPI = parser.readLocation("integerDefault.yaml", null, options).getOpenAPI();
+        Assert.assertNotNull(openAPI);
+        Assert.assertFalse (((IntegerSchema) openAPI.getPaths().get("/fileUpload").getPost().getRequestBody().getContent().get("multipart/form-data").getSchema().getProperties().get("intMetadata")).getFormat() == "int32");
+        Assert.assertNull ( openAPI.getPaths().get("/mockResponses/primitiveDoubleResponse").getGet().getResponses().get("200").getContent().get("application/json").getSchema().getFormat());
+        Assert.assertNull (openAPI.getPaths().get("/primitiveBody/binary").getPost().getRequestBody().getContent().get("application/octet-stream").getSchema().getFormat());
+
+
+    }
+
```
Comment (frantuma) [inline modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java]: testing `type: integer, format: int16` and/or other cases present in test yaml would be good

File: modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import io.swagger.v3.oas.models.parameters.Parameter;
 import io.swagger.v3.oas.models.parameters.RequestBody;
 
+import io.swagger.v3.parser.OpenAPIV3Parser;
```
Comment (frantuma) [inline modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java]: what is this change for? possibly revert changes for this file

File: modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java [hunk 3]
```diff
@@ -25,9 +26,11 @@
 
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
 import static org.testng.Assert.assertTrue;
 
 public class OpenAPIParserTest {
+
```
Comment (frantuma) [inline modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java]: what is this change for? possibly revert changes for this file

File: modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java (hunk 2 above)
Comment (gracekarina) [reply modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java]: I'll remove

File: modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java (hunk 3 above)
Comment (gracekarina) [reply modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java]: yes I will

**Pull request merged**